README.md

```diff
@@ -599,7 +599,19 @@ a chat service, sometimes it takes several hours for community members to respon
 — please be patient.
 
 ### Credits
-Props: [@danlouw](https://github.com/danlouw), [@jeffpaul](https://github.com/jeffpaul), [@rheinardkorf](https://github.com/rheinardkorf), [@valendesigns](https://github.com/valendesigns)
+Props:  
+[Daniel Louw (@danlouw)](https://github.com/danlouw)  
```

Comment:
Let's remove the line breaks, and also add Luke and Brendan.

Reply:
Was going to ask. Luke had a PR that in the old repo that never merged. Meaning he did work. Will update in the morning!